setup/apache.yml

```diff
@@ -18,7 +18,7 @@ apache_cosign_deny_friend:     "global=app_cosign_deny_friend default=no"
 
 apache_static_path:
   global: deploy_root
-  output: File.join(deploy_root, app_name, "shared", "public")
+  output: File.join(deploy_root, app_name, "current", "public")
```

Comment:
This was affecting their need to serve the static jekyll site - that content is kept in their git repository and deployed to `current/public`.